src/main/kotlin/io/zerobase/smarttracing/features/devices/DevicesResource.kt

```diff
@@ -18,6 +20,12 @@ data class CreateCheckInRequest(
     val type: ScanType,
     val location: Location?
 )
+
+data class SelfReportedTestResult(
```

Comment:
local date and time makes sense in that you could give location context when searching / sorting self reported test results instead of converting everything to the same time zone. We'd probably look at geographic clusters and can convert to get everything on a similar time scale using local time, date with location. It looks like timezone.instant helps give some semblance of timezone difference when invoking to string

Reply:
realistically, we don't really know what timezone the person self-reporting is in. we can only guess at a device's location based on the check-in locations, but even then, there are plenty of places where timezones change in small geographic areas. 

based on my understanding of what we are wanting to do and what data we are collecting, i don't think the time portion would be useful. it would all have to be at midnight anyway, and most likely we would just do UTC. if we are going to force everything to the same time in the same timezone, a local date (which is basically the calendar date only) gives us all the precision we need - especially since i'm pretty sure the UI will present a calendar widget for the use to select from.